test-plan.md

```diff
@@ -93,6 +93,38 @@ The easist way to verify that a project was successfully loaded is to open a .cs
 * In a project that uses globbing (.NET Core), use the VS Code file explorer to add a new file next to the csproj. Intellisense/sighelp/etc should be available in the new file
 * Add a new file and reference a type in it from a different file. Deleting from disk the file containing the referenced type  should produce error messages
 
+#### Razor
+The Razor experience is available when you open a .cshtml file in a valid OmniSharp project.
+
+##### C# Completion
+* Typing `@DateTime.Now` and `@(DateTime.Now)` provides completions throughout typing.
+* Completion is available for types that exist in the project (i.e. `Program`)
+* Typing `@model DateTime` prompts for completion for the `model` symbol and the `DateTime` symbol.
+
+#### C# Signature Help
+* Typing `@Html.Raw()` prompts for signature help inside of the `()`.
+
+#### C# Diagnostics
+* Typing `@ThisDoesNotExist` results in an error being created and squiggled in the .cshtml file. NOTE: This error squiggly will be misaligned due to known issues.
+
+#### Razor Options
+
+##### razor.disabled
+This option can be set to `true` to disable the above described C# experience.
+
+##### razor.languageServer.directory
+Setting this path to any of the values as listed below, should start the Razor Language server.
+  * undefined - Razor Language server must start using the copy of Razor Language server shipped with the extension.
+  * Some absolute path to a directory - Razor Language server must start using the local copy of Razor Language server that exists in the path pointed to by the config.
+  Example:C:/Razor.VSCode/artifacts/build/win-x64/ 
+
+##### razor.languageServer.trace
+This option should always be displayed in the View --> Output --> Razor Log window
+This option can be set to any of the following values:
+  * "Off" - Will launch Razor Language server with its log output set to 'Off'.
```

Comment:
Can we say something like "If the option is set to "Off", nothing will show up or something ?

Reply:
Content still shows up, you just don't get as verbose of logs. I'll clarify